notebooks/community/tensorboard/get_started.ipynb

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
<p>Can you either update the correct link? or remove the View on Github? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>removed</p>